doc/frameworks/xgboost/using_xgboost.rst

```diff
@@ -390,6 +390,57 @@ The function should return a byte array of data serialized to ``content_type``.
 The default implementation expects ``prediction`` to be a NumPy array and can serialize the result to JSON, CSV, or NPY.
 It accepts response content types of "application/json", "text/csv", and "application/x-npy".
 
+Bring your own model
+--------------------
+
+You can deploy an XGBoost model that you trained outside of SageMaker by using the Amazon SageMaker XGBoost container.
+Typically, you save an XGBoost model by pickling the ``Booster`` object or calling ``booster.save_model``.
+The XGBoost `built-in algorithm mode <https://docs.aws.amazon.com/sagemaker/latest/dg/xgboost.html#xgboost-modes>`_
+supports both a pickled ``Booster`` object and a model produced by ``booster.save_model``.
+For a sample notebook that shows to use the XGBoost built-in algorith mode to load a pre-existing XGBoost model,
+see `Amazon SageMaker XGBoost Bring Your Own Model <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/xgboost_bring_your_own_model/xgboost_bring_your_own_model.ipynb>`_.
```

Comment:
typo with "algorithm". also, would it make more sense to put this in the "Use XGBoost as a Built-in Algorithm" section?

```suggestion
For a sample notebook that shows to use the XGBoost built-in algorithm mode to load a pre-existing XGBoost model,
see `Amazon SageMaker XGBoost Bring Your Own Model <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/xgboost_bring_your_own_model/xgboost_bring_your_own_model.ipynb>`_.
```

Reply:
Thanks for the suggestion. There will also be a section on AWS documentation, and considering your suggestion, I think it makes more sense to move this to AWS documentation since the notebook is about the built-in mode. Python SDK doc can focus on script mode.

Reply:
+1